data/data.yaml

```
@@ -503,3 +503,119 @@
+    Сложные случаи:
+    1. Если пустое слово принадлежит L(r_1), то частичная производная от r_1 r_2 по c получается путем взятия частичных производных от r_1 и r_2 по c и конкатенацией результата со вторым регулярным выражением r_2.
+    2. Если пустое слово не принадлежит L(r_1), то частичная производная от r_1 r_2 по c получается путем взятия частичной производной от r_1 по c и конкатенацией результата со вторым регулярным выражением r_2.
+    3. Частичная производная от r_1 | r_2 по c является объединением частичных производных от r_1 и r_2 по c.
+    4. Частичная производная от r* по c состоит из всех конкатенаций r'r*, где r' - это частичная производная от r по c.
+    Полученные частичные производные используются для построения автомата Антимирова, где состояния соответствуют этим частичным производным.
+  author: Егорова Алиса
+- question: >
+    Какие условия необходимы для бисимуляции двух НКА?
+  answer: >
+    Бисимуляция определяет, когда два автомата ведут себя одинаково с точки зрения наблюдателя, который может видеть только входные символы и выходные состояния.
+    Чтобы определить отношение бисимуляции на конечных автоматах, к отношению бисимуляции на LTS(labelled transition system - тройка {S,∑,Q}, где S,∑,Q - множества состояний, меток и переходов соответсвенно) нужно добавить ограничения на бисимуляцию начальных и конечных состояний. Более точно, для бисимуляции НКА A1 и A2 необходимы следующие условия: 
+    1) каждому состоянию A1 бисимилярно состояние A2 и наоборот 
+    2) стартовому состоянию A1 бисимилярно стартовое состояние A2 
+    2) каждому финальному состоянию A1 бисимилярно финальное состояние A2 и наоборот 
+    Бисимилярные НКА распознают равные языки.
+    Состояния q_i и q_j в НКА А бисимирлярны (q_i ~ по A q_j), если они связаны LTS-бисимуляцией и имеют одинаковую финальность в А.
+    Бисимуляцию НКА можно переформулировать как отношение бисимуляции состояний НКА такое, что стартовые состояния бисимилярны.
+    Отношение бисимуляции имеет важное свойство: бисимилярные состояния в автомате можно объединить без изменения его семантики. Это преобразование часто позволяет существенно упростить НКА.
+  author: Егорова Алиса
+- question: >
+    Являются ли финальные состояния и состояния-ловушки бисимилярными?
+  answer: >
+    Состояния q_i и q_j в НКА А бисимирлярны (q_i ~ по A  q_j), если они связаны LTS-бисимуляцией и имеют одинаковую финальность в А.
+    Все состояния-ловушки в любом полном автомате (т.е. с явно присутствующими переходами по всем буквам алфавита) бисимилярны друг другу. Все финальные состояния без переходов из них (кроме как в ловушки) также бисимилярны.
+  author: Егорова Алиса
+- question: >
+    Что такое трансформационный моноид?
+  answer: >
+    Трансформационный моноид M_A для ДКА А - это моноид функций F_a таких, что F_a(q_i) = q_j тогда и только тогда, когда q_i -> q_j по a в А, где q_i и q_j - состояния. 
```

```suggestion
    Трансформационный моноид M_A для ДКА А - это моноид функций F_a таких, что F_a(q_i) = q_j тогда и только тогда, когда есть переход из состояния q_i в состояние q_j по символу a в А.
```

и добавь про базис функций